evennia/commands/default/help.py

```diff
@@ -487,16 +487,36 @@ def do_search(self, query, entries, search_fields=None):
                 {"field_name": "tags", "boost": 1},  # tags are not used by default
             ]
         match, suggestions = None, None
+        base_query = query.lstrip("@")
```

Comment:
This is a good idea, but this is too specific since `@` is not the only possible prefix understood (and ignored) by the command parser. The understood prefixes are given by `settings.CMD_IGNORE_PREFIXES`.

Reply:
I've added support for `settings.CMD_IGNORE_PREFIXES`.